src/components/filters/AttributeFilter/AttributeFilter.tsx

```diff
@@ -85,36 +92,94 @@ export class AttributeFilter extends React.PureComponent<IAttributeFilterProps>
     }
 
     public render() {
-        const { locale, projectId, uri, identifier } = this.props;
+        const { locale, projectId } = this.props;
         const { md } = this.sdk;
+        const { identifier, uri } = this.getIdentifierOrUri();
+
         return (
             <IntlWrapper locale={locale}>
                 <AttributeLoader uri={uri} identifier={identifier} projectId={projectId} metadata={md}>
-                    {props => this.renderContent(props)}
+                    {(props: IAttributeLoaderChildrenProps) => this.renderContent(props)}
                 </AttributeLoader>
             </IntlWrapper>
         );
     }
 
+    private isInverted() {
+        const { filter } = this.props;
+
+        return !AFM.isPositiveAttributeFilter(filter);
+    }
+
+    private getIdentifierOrUri() {
+        const { filter, identifier, uri } = this.props;
+        if ((filter && identifier) || (filter && uri) || (identifier && uri)) {
```

Comment:
Other way to do this that doesn't grow exponentially :-D
if(Number(!!filter) + Number(!!identifier) + Number(!!uri) > 1)

Reply:
Also, you already show a warning when the identifier or uri is used, so don't need to show another warning I guess.

Reply:
I hope that if we deprecate old properties that we will not add another new ones to grow the condition